### DISCUSSION TITLE
Make it where the user doesn't have root/sudo access

### DISCUSSION THREAD

**Discussion by Darker-Ink** [coder/code-server#4130] [General]
So I want to install a code server for my friend but he doesn't want to have sudo access how would I do this for him?

**Comment by lg-nineoneninethreenine** [comment]
You just need to use the curl link and you have code-server installed.

**Reply by Darker-Ink** [reply]
I have the code server install. The problem his my friend has sudo access

**Reply by Darker-Ink** [reply]
Since he is logged in as root

**Reply by lg-nineoneninethreenine** [reply]
Then just run it! Root is a permanent sudo user, you can run apt update or any updating of packages, installing, in fact, everything without the sudo command as root.

**Reply by Darker-Ink** [reply]
> Then just run it! Root is a permanent sudo user, you can run apt update or any updating of packages, installing, in fact, everything without the sudo command as root.

I Don't want him to have Sudo. That was the legit the question. This doesn't solve how to let him not have sudo access **I Do Not Want Him To Have Sudo Access**

**Reply by lg-nineoneninethreenine** [reply]
Then why not make a new user with the `adduser` command?

**Reply by lg-nineoneninethreenine** [reply]
And then remove hin from sudo group _after_ installing code-server?
